tests/frontend/dismissable_controller_test.js

```diff
@@ -40,13 +40,13 @@ describe("Dismissable controller", () => {
     application.register("dismissable", DismissableController);
   });
 
-  describe("no cookie is present", function() {
-    it("the element is not dismissed", function() {
+  describe("no cookie is present", function () {
```

Comment:
Do you have a different formatter running on js code?

Reply:
Hmm, I think this got reflowed by whatever VS Code's default is. I can revert these changes, if you'd like.

Reply:
Not a problem, especially since I don't think our current eslint rule sets one or the other. These **are** anonymous functions, so it's a little clearer.
Looks like [the rule was deprecated](https://eslint.org/docs/latest/rules/space-before-function-paren#:~:text=This%20rule%20was%20deprecated%20in%20ESLint%20v8.53.0.%20Please%20use%20the%20corresponding%20rule%20in%20%40stylistic/eslint%2Dplugin%2Djs.) (as most style rules have), so we probably need to revisit the js linting vs formatting - I'll file an issue. No change here necessary.